### DISCUSSION TITLE
How possible store in Column field some custom data?

### DISCUSSION THREAD

**Discussion by MatsiukMykola** [sqlalchemy/sqlalchemy#9787] [Usage Questions] [answered]
I will store some additional metadata information inside the Column object
```python
class ProductModel(Base, Timestamp):
    product_price = Column(Integer, nullable=True, comment="Product duration", my_custom_data = {'pydantic_type': condecimal(decimal_places=2, gt=0)})
```

for example, there I will store UI information for Pydantic

SQLModel - bad choice, have too many bugs (((
issue - only how store? doc property support only str

**Comment by CaselIT** [comment, marked as answer]
Hi,

You can use info to store any custom data. https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.Column.info

Many sqlalchemy object have a similar info dict that can be used to store application metadata
